src/Frame.php

```diff
@@ -92,7 +92,7 @@ public function __construct(?string $functionName, string $file, int $line, ?str
         $this->file = $file;
         $this->line = $line;
         $this->rawFunctionName = $rawFunctionName;
-        $this->absoluteFilePath = $absoluteFilePath;
+        $this->absoluteFilePath = $absoluteFilePath ?? $file;
```

Comment:
In case the frame is internal then `$file` would be set to `[internal]` because we need a placeholder somehow but reporting that value as absolute path of the file would not be correct